tutorials/R/R-skills/Using-hdf5-r/Subset-HDF5-hyperspectral/Subset-HDF5-hyperspectral.Rmd

```diff
@@ -322,7 +322,7 @@ it is weird that `NULL` indicates "all" in this circumstance, but that is the
 default behavior for this, and many other, functions).
 
 
-```{r plot-example-band}
+```{r plot-example-band, fig.cap="Hyperspectral reflectance plot"}
```

Comment:
Hi @adernbach , could you please revise these captions to be a bit more descriptive for people who aren't able to view the plot? Something like "aerial image of a highschool including buildings, parking lot, and athletic facilities", with the inset more like "a single quadrant of the image shown above, this time we can only see the baseball diamond and surrounding forest"